yt_dlp/extractor/youtube.py

```diff
@@ -4573,7 +4573,7 @@ def is_bad_format(fmt):
             'live_status': live_status,
             'release_timestamp': live_start_time,
             '_format_sort_fields': (  # source_preference is lower for potentially damaged formats
-                'quality', 'res', 'fps', 'hdr:12', 'source', 'vcodec:vp9.2', 'channels', 'acodec', 'lang', 'proto'),
+                'quality', 'res', 'fps', 'hdr:12', 'source', 'vcodec', 'channels', 'acodec', 'lang', 'proto'),
```

Comment:
the compat opt won't have any effect on youtube?

Reply:
No, they will. The user provided sort order takes priority over the extractor provided order.